### DISCUSSION TITLE
WalletConnect not opening in iPhone

### DISCUSSION THREAD

**Discussion by callmejm** [WalletConnect/walletconnect-monorepo#2311] [Developer Support] [answered]
I using Rainbowkit, while connect wallet what I try to do is select Wallet Connect > Browse TokenPocket, then I facing this issue. Clicked Allow but nothing happen 
This website is repeatedly trying to open another application.

![File_000 (1)](https://user-images.githubusercontent.com/132695402/236686267-a4059de3-79ba-4b97-9a1a-4eeeeae93c60.png)

**Comment by finessevanes** [comment]
@callmejm are you building a dapp and experiencing this issue or are you on a website that uses WallletConnect and you're unable to connect to TokenPocket?

If it's the latter, please provide the URL so we can investigate.

**Reply by callmejm** [reply]
Hi thanks for reply, I building a dapp and facing this issue. Sorry that I cannot provide any link yet

**Reply by finessevanes** [reply, marked as answer]
If you're building your dapp, I would suggest to use Web3Modal vs using RK as our modal will have the most up-to-date packages.
https://docs.walletconnect.com/2.0/web3modal/about
